src/diffusers/pipelines/aura_flow/pipeline_aura_flow.py

```diff
@@ -634,7 +669,16 @@ def __call__(
         # Offload all models
         self.maybe_free_model_hooks()
 
+        if self.text_encoder is not None:
+            if isinstance(self, AuraFlowLoraLoaderMixin) and USE_PEFT_BACKEND:
+                # Retrieve the original scale by scaling back the LoRA layers
+                unscale_lora_layers(self.text_encoder, lora_scale)
+
         if not return_dict:
             return (image,)
 
         return ImagePipelineOutput(images=image)
+
+    @property
+    def attention_kwargs(self):
+        return self._attention_kwargs
```

Comment:
This should be positioned upper.

Reply:
Done in https://github.com/huggingface/diffusers/pull/10216/commits/3b9e65543a080ab4eb978fe7c9c7f4d0198b1139